pages/agent/v3/help/_env_unset.md

```diff
@@ -21,26 +21,26 @@ Unsets environment variables in the current job execution environment.
 Changes to the job environment variables only apply to subsequent phases of the job.
 This command cannot unset Buildkite read-only variables.
 
-To read the new values of variables from within the current phase, use ′env get′.
+To read the new values of variables from within the current phase, use `env get`.
 
 Note that this subcommand is only available from within the job executor with the job-api experiment enabled.
 
 ### Examples
 
-Unsetting the variables `LLAMA` and `ALPACA`:
+Unsetting the variables `LLAMA` and `ALPACA`)
 
-```
+```shell
 $ buildkite-agent env unset LLAMA ALPACA
-Un-set:
+Unset:
```

Comment:
Given the command is `buildkite-agent env unset` I think we need to make a consistent (non) use of hypens.